### ISSUE TITLE
Support for MathSpeak "Semantic Interpretation" or "Explicivity" settings?

### ISSUE THREAD

**Issue by brichwin** [Speech-Rule-Engine/speech-rule-engine#668]
Are there any plans to support the MathSpeak "Semantic Interpretation" or "Explicivity" settings? (as described here: https://www.seewritehear.com/accessible-mathml/mathspeak/examples/settings-tutorial/ )

The medium and high semantic interpretation settings might help users adopt MathSpeak that normally prefer ClearSpeak's more natural sounding speech. It seems that MathSpeak can handle higher levels of math than the ClearSpeak that is currently being generated can. 

The medium and high explicivity settings would be helpful to non-visual listeners that are new to listening to a math speech grammar. This is especially relevant to newly visually impaired students that must make it through a general education (or other) required math course.

**Comment by zorkow** [comment]
I agree that MathSpeak is better for disambiguierung higher Mathematics. But I never felt that the "semantics" and "explicitivity" ideas are thought out enough to implement. Some of the examples seem to be even contradictory: E.g., the matrix. 

Currently, SRE tries to strike some balance between too explicit and not explicit enough. For example it would always say 
`4 and three-fifths equals StartFraction 23 Over 5 EndFraction` rather than expressing the vulgar fraction more verbosely.

But I can see that it might be helpful for people to get information what layer is closed instead of simply `Baseline` or `EndFraction`. For which expressions do you think those considerations would be particularly relevant?

One thing I always shied away from is the use of "quantity" as this seems to be very US specific (a bit like saying "negative x" for `-x`). I am also not sure if the article "The " adds much in "the 3by3 matrix". Clearspeak has that, which makes it very awkward during localisation. 
For example, the last example on the page has `la fracción...`. Now the person who did our Spanish localisation was very emphatic that one would not use this. Of course, that was one Math teacher from Spain, only...
